README.md

```diff
@@ -11,9 +11,17 @@ Please read the wiki for more details about checks already implemented.
 
 ## Dependencies
 
+### Boost
+
 The dependencies are the Boost library core, and its submodules: Boost.filesystem, Boost.program_options, Boost.regex and Boost.system.
 If you are using Ubuntu, the required packages names will be `libboost-dev`, `libboost-filesystem-dev`, `libboost-program-options-dev` and `libboost-regex-dev`.
 
+### ODB
```

Comment:
Mention that the SQLite library is a dependency and the required packages in Ubuntu are `libsqlite3-0` and `libsqlite3-dev`.